pebbles/tasks/celery_app.py

```diff
@@ -50,17 +49,13 @@ def do_post(token, api_path, data):
     return resp
 
 
-def get_config():
-    """
-    Retrieve dynamic config over ReST API. Config object from Flask is unable to resolve variables from
-    database if containers are used. In order to use the ReST API some configuration items
-    (Variable.filtered_variables) are required. These are read from Flask config object, as these values
-    cannot be modified during the runtime.
+def get_dynamic_config():
+
+    """ Get the dynamic config by using the config YAML file, default values in config.py,
```

Comment:
Correct me if i'm wrong but doesn't BaseConfig also check the environment and those precede even config YAML?